### ISSUE TITLE
Low Battery Reporting as Bypass

### ISSUE THREAD

**Issue by amattas** [amattas/hass-dmp#18]
Sensors with low battery are reporting a status of Bypass which is incorrect unless they've been explicitly bypassed.

**Comment by NebulaGazer** [comment]
Agreed - I have this in a local bugfix branch but have some other branches that need to be merged first. Would like to make sure your other bug reports are resolved first.
I think this just needs to change from 'Bypass' to 'Low Power', but I fixed the wireless device with low power and could not test.

https://github.com/amattas/hass-dmp/blob/master/custom_components/dmp/__init__.py#L281

**Comment by NebulaGazer** [comment]
https://github.com/amattas/hass-dmp/pull/36
